components/newHome/Services/Services.tsx

```diff
@@ -0,0 +1,275 @@
+import classNames from "classnames/bind";
+import { useNavigatorScroll } from "../../Home/scroll";
+import Image from "next/image";
+import foregroundBorder from "../../../public/static/images/newHome/progress_foreground_border.svg";
+import styles from "./Services.module.scss";
+import { useEffect, useState } from "react";
+
+const cx = classNames.bind(styles);
+
+type TserviceData = {
+  // TODO: enum으로
+  serviceName: string;
+  serviceLink: string;
+  serviceLogo: string;
+  serviceLogoAltMsg: string;
+  servicePhoneImg: string;
+  servicePhoneImgAltMsg: string;
+  serviceNameForServiceNameH1Tag: string;
+  serviceDescription: string;
+};
+
+const serviceDataArray: TserviceData[] = [
+  {
+    serviceName: "snutt",
+    serviceLink: "https://wafflestudio.com",
+    serviceLogo: "/static/images/logo/snutt_logo.svg",
+    serviceLogoAltMsg: "snutt app logo",
+    servicePhoneImg: "/static/images/newHome/phone_snutt.svg",
+    servicePhoneImgAltMsg: "snutt phone display image",
+    serviceNameForServiceNameH1Tag: "SNUTT",
+    serviceDescription: "서울대학교 시간표 어플",
+  },
+  {
+    serviceName: "siksha",
+    serviceLink: "https://wafflestudio.com",
+    serviceLogo: "/static/images/logo/siksha_logo.svg",
+    serviceLogoAltMsg: "siksha app logo",
+    servicePhoneImg: "/static/images/newHome/phone_siksha.svg",
+    servicePhoneImgAltMsg: "siksha phone display image",
+    serviceNameForServiceNameH1Tag: "식샤",
+    serviceDescription: "서울대학교 학식 어플",
+  },
+  {
+    serviceName: "snuboard",
+    serviceLink: "https://wafflestudio.com",
+    serviceLogo: "/static/images/logo/snuboard_logo.svg",
+    serviceLogoAltMsg: "snuboard app logo",
+    servicePhoneImg: "/static/images/newHome/phone_snuboard.svg",
+    servicePhoneImgAltMsg: "snuboard phone display image",
+    serviceNameForServiceNameH1Tag: "스누보드",
+    serviceDescription: "서울대학교 과별 알림 어플",
+  },
+];
+
+const serviceDataArrayLength: number = serviceDataArray.length;
+
+function ProgressBox({
+  serviceIndex,
+  increaseIndex,
+  isPlaying,
+  toggleIsPlaying,
+}: {
+  serviceIndex: number;
+  increaseIndex: () => void;
+  isPlaying: boolean;
+  toggleIsPlaying: () => void;
+}) {
+  const [timeCount, setTimeCount] = useState<number>(0);
+
+  useEffect(() => {
+    if (timeCount === 3000) {
+      setTimeCount(0);
+      increaseIndex();
+    }
+  }, [timeCount, increaseIndex]);
+
+  useEffect(() => {
+    if (isPlaying) {
+      const interval = setInterval(() => {
+        setTimeCount((prev) => prev + 5);
+      }, 5);
+      return () => {
+        clearInterval(interval);
+      };
+    }
```

Comment:
이 부분에서 연산량이 많아서 로딩(회전) 애니메이션이랑 넘어가는 애니메이션 타이밍이 안 맞는 것 같아요
차라리 isPlaying이 해제되면 회전량을 초기화시키고 여기서는 3초에 한번씩 인덱스 증가시키는 걸로 바꾸면 어떨까요?
```suggestion
    if (isPlaying) {
      const interval = setInterval(() => {
        increaseIndex();
      }, 3000);
      return () => {
        clearInterval(interval);
      };
    }
```

Reply:
일단 단위 5=> 50으로 수정했습니다. 다시한번 확인해주실 수 있을까요? 연산량이 많은 게 문제되면 말씀해주신 대로 바꾸는 게 나을 듯